src/ajax/map_variants.php

```diff
@@ -228,11 +227,11 @@ function lovd_mapVariantToTranscripts (&$aVariant, $aTranscripts)
 
             $aVariantIDs = $_DB->query(
                 'SELECT id FROM ' . TABLE_VARIANTS .
-                ' WHERE chromosome = ' . $sChr . // Only taking our picked chromosome
-                '    AND ' . $aGBColumns['DNA'] . '= ""' . // DNA field of this GB is empty
-                '    AND mapping_flags & ' . MAPPING_ALLOW . // Mapping  is allowed
+                ' WHERE chromosome = ?' . // Only taking our picked chromosome
+                '    AND (' . $aGBColumns['DNA'] . ' = "" OR ISNULL(' . $aGBColumns['DNA'] . '))' . // DNA field is empty
```

Comment:
In LOVD we always use `column IS NULL` instead of `ISNULL(column)`.
Also, note that `$aGBColumns['DNA']` will need some quoting as MySQL doesn't like column names with slashes. Check other queries to see how that's done.

Reply:
Noted and implemented!